package.json

```diff
@@ -89,5 +92,9 @@
     "developer experience",
     "dxheroes",
     "code analysis"
-  ]
+  ],
+  "jest": {
```

Comment:
Is it necessary to have this config here if we have `jest.config.js` already?

Reply:
It shouldn't, I'll remove it.